db/migrate/20150906173134_create_teams.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTeams < ActiveRecord::Migration
+  def change
+    create_table :teams do |t|
+      t.belongs_to :account
+      t.belongs_to :team
```

Comment:
cool. do you need to add `foreign_key: :team_id` to those associations? I think that by default it assumes the foreign key is based on the association name.